torchdata/nodes/prefetch.py

```diff
@@ -14,13 +14,23 @@
 
 
 class Prefetcher(BaseNode[T]):
+    """Prefetcher is a node that prefetches data from the source node and stores it in a queue.
+
+    Parameters:
+        source (BaseNode[T]): The source node to prefetch data from.
+        prefetch_factor (int): The number of batches to prefetch ahead of time.
```

Comment:
Shall we say "The number of ~batches~ items to prefetch"?
Because we can also use it on a single node.